ssl/statem/statem_clnt.c

```diff
@@ -1122,6 +1122,9 @@ MSG_PROCESS_RETURN tls_process_server_hello(SSL *s, PACKET *pkt)
          * overwritten if the server refuses resumption.
          */
         if (s->session->session_id_length > 0) {
+#ifndef OPENSSL_NO_AKAMAI
```

Comment:
Don't think this is required, no? A lot of these weird defines aren't even documented so we might as well drop them.